manifests/08_service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: marketplace-operator
```

Comment:
Is this the best name for this service? Can we be slightly more descriptive here?

Reply:
We can call it `marketplace-operator-metrics` if you'd like.

Reply:
Other second level operators also expose metrics, I wonder if everyone follows the same naming convention here :)

Reply:
Though the service monitor just uses the `olm-operator` name: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/manifests/0000_90_olm_00-service-monitor.yaml#L5